_data/changelogs/component-pagination.yml

```diff
@@ -2,6 +2,15 @@ title: Pagination
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Updated text-decoration style on pagination links so page numbers are underlined.
+  summaryAdditional: Users should confirm the pagination page numbers are underlined.
+  isBreaking: true
```

Comment:
You have this listed as a non-breaking change in #5970. Since users will not be required to make any updates manually, I'd agree this is a non-breaking change!

```suggestion

```

Reply:
Resolved in [009e774](https://github.com/uswds/uswds-site/pull/2793/commits/009e774d0d925b3f040a32bc9888e27fb2394fc6)